python/lsst/daf/base/exampleLib.py

```diff
@@ -0,0 +1,12 @@
+from _exampleLib import DateTime
+
+def toPython(self, timescale=None):
+    """Convert a DateTime to Python's datetime
+
+    @param timescale  Timescale for resultant datetime
+    """
+    import datetime
+    nsecs = self.nsecs(timescale) if timescale is not None else self.nsecs()
+    return datetime.datetime.utcfromtimestamp(nsecs/10**9)
```

Comment:
When the time comes to do this properly we should just use `astropy.time` and https://github.com/lsst-dm/python-experiments/blob/master/lsstx/DateTime.py will be a useful resource.